app/controllers/blogs_controller.rb

```diff
@@ -42,23 +38,22 @@ def update
   end
 
   def destroy
-    raise ActiveRecord::RecordNotFound unless current_user == @blog.user
-
     @blog.destroy!
     redirect_to blogs_url, notice: 'Blog was successfully destroyed.', status: :see_other
   end
 
   private
 
   def set_blog
-    @blog = Blog.find(params[:id])
+    @blog = current_user.blogs.find(params[:id])
   end
 
   def blog_params
+    default_permit_columns = :title, :content, :secret
```

Comment:
```suggestion
    default_permit_columns = [:title, :content, :secret]
```

配列の`[]`を明示的に書くことの方が多いと思いますー